http/1.7/modules/ROOT/pages/http-polling-source-ref.adoc

```diff
@@ -0,0 +1,300 @@
+</flow>
+----
+
+Then, the corresponding expression to split the array is `#[payload]`. In the splitting expression, the referred `payload` is the entire response already treated as a JSON file. Meanwhile, the `payload` referred in the `logger` processor refers to the corresponding payload for its execution, on one execution `{'name': 'Mondi'}`, on the other execution `{'name': 'Viena'}`). The `splitExpression` configuration looks like this:
+
+[source,xml,linenums]
+----
+<flow name="jsonPollingFlow">
+    <http:polling-source config-ref="requestConfig" path="json" splitExpression="#[payload]">
+        <scheduling-strategy>
+            <fixed-frequency frequency="10" timeUnit="SECONDS"/>
+        </scheduling-strategy>
+    </http:polling-source>
+    <logger message="#[payload]"/>
+</flow>
+----
+
+The following example shows two concurrent executions of the flow after one same polling, one execution with `<name>Mondi</name>` and another execution with `<name>Viena</name>`. After 10 hours, the polling is executed again. In this example, the server returns the same result:
+
+[source,xml,linenums]
+----
+<flow name="xmlListenerFlow">
+    <http:listener config-ref="listenerConfig" path="/xml"/>
+    <set-payload value="#[output application/xml --- {'something': {'element': [{name: 'Mondi'}, {name: 'Viena'}]}}]"/>
+</flow>
+
+<flow name="xmlPollingFlow">
+    <http:polling-source config-ref="requestConfig" path="xml"
+        splitExpression="#[payload.something.*element]">
+        <scheduling-strategy>
+            <fixed-frequency frequency="10" timeUnit="HOURS"/>
+        </scheduling-strategy>
+    </http:polling-source>
+    <logger message="#[payload]"/>
+</flow>
+----
+
+== Configure Watermarking Expression
+
+In the previous simple examples, the server always returned the same HTTP response. In more complex scenarios the server needs to know that it has to send the next response. You can send headers, URI parameters, body, or query parameters, but if these parameters always contain the same values in the HTTP request, the server does not know what would be the next response.
+
+For these polling scenarios, you use watermarking, where the server returns a watermark value on the payload itself, or in every item. For example, the watermark value can be a timestamp that can refer to the entire collection, or every item could have its own timestamp. In any case, you provide a watermark expression using the `watermark` placeholder.
+
+The expression extracts the watermark from the response, and subsequently, use this watermark to send the requests to the server. You can use expressions for the body, headers, URI parameters and query parameters' values in the the `watermark` placeholder.
+
+Note that in the first execution, the `watermark` placeholder value is `null`, which you might want to consider in the server or the expression where the placeholder is used.
```

Comment:
I don't understand what this means: is the value on first use always `null`, and what are you recommending that the audience do about that, if anything? Can the user circumvent this or does the user just need to be aware of it?

Reply:
Be sure that this is consistent with your example, Line 151.